hw/ip/flash_ctrl/data/flash_ctrl.sv.tpl

```diff
@@ -548,15 +556,17 @@ module flash_ctrl
     .en_ifetch_i (prim_mubi_pkg::MuBi4False),
     .req_o       (rd_fifo_ren),
     .req_type_o  (),
-    .gnt_i       (rd_fifo_rvalid),
+    // if there is no valid read operation, don't hang the
+    // bus, just let things normally return
```

Comment:
what data is returned in this case? is the last-read data still shown? in that case we may have to blank the data to zero...

Reply:
yeah i thought about that, but i don't really think it's a big deal in this case. Since the error is shown to the host already. (I did originally blank it to all 1's, the flash erased state).

Reply:
Oh so you mean that over reading cannot go undetected either way?
Yeah you are probably right...